config/routes.rb

```diff
@@ -1,4 +1,5 @@
 Rails.application.routes.draw do
+  get "sessions/new"

```

Comment:
sao cái này lại ném ngoài scope thế nhỉ ?

Reply:
cái đấy k cần dùng nữa nên mình xóa đi rồi. vì ở dưới mình đã định nghĩa router đấy rồi
get "/login", to: "sessions#new"